include/nbl/video/SPhysicalDeviceFeatures.h

```diff
@@ -621,9 +624,148 @@ struct SPhysicalDeviceFeatures
     
     /* Extensions Exposed as Features: */
 
+    /* VK_AMD_buffer_marker */
+    /* VK_AMD_calibrated_timestamps */
+    /* VK_AMD_display_native_hdr */
+    /* VK_AMD_draw_indirect_count */
+    /* VK_AMD_gcn_shader */
+    /* VK_AMD_gpa_interface */
+    /* VK_AMD_gpu_shader_half_float */
+    /* VK_AMD_gpu_shader_half_float_fetch */
+    /* VK_AMD_gpu_shader_int16 */
+    /* VK_AMD_image_layout_resolve */
+    /* VK_AMD_memory_overallocation_behavior */
+    /* VK_AMD_mixed_attachment_samples */
+    /* VK_AMD_negative_viewport_height */
+    /* VK_AMD_pipeline_compiler_control */
+    /* VK_AMD_programmable_sample_locations */
+    /* VK_AMD_rasterization_order */
+    /* VK_AMD_shader_ballot */
```

Comment:
always enable, don't report as a limit, have it contribute to `shaderSubgroup` reporting instead (it provides a subset of the functionality).

Reply:
GL has identical named extension btw